javascript/MLE-Demo/cleanup.sql

```diff
@@ -0,0 +1,5 @@
+BEGIN
+  EXECUTE IMMEDIATE 'DROP PROCEDURE SET_WALLET';
+  EXECUTE IMMEDIATE 'DROP TABLE INVENTORY';
+  EXCEPTION WHEN OTHERS THEN NULL;
```

Comment:
This will swallow any error, even valid ones that the user may need to act upon.
Please remove the `WHEN OTHER THEN NULL` and only catch and ignore errors that are expected.